src/components/v5/common/Fields/Select/Select.module.css

```diff
@@ -3,17 +3,17 @@
 }
 
 .button {
-  @apply w-full pl-3.5 pr-4 py-3 rounded bg-base-white flex justify-between items-center text-md font-normal transition-all duration-normal;
+  @apply w-full pl-3.5 pr-4 py-3 rounded bg-base-white flex justify-between items-center text-md font-normal transition-all duration-normal border;
 }
 
 .options {
   @apply border-gray-200 hidden;
 }
 
 .show {
-  @apply flex flex-col py-2 px-3.5 border border-gray-300 rounded relative left-0 z-[1] bg-base-white w-full mt-1 max-h-[12.5rem] overflow-x-scroll;
+  @apply flex flex-col py-2 px-2 border border-gray-300 rounded relative left-0 z-[1] bg-base-white w-full mt-1 max-h-[12.5rem] overflow-x-auto;
```

Comment:
I think we have to remove this style overflow-x-auto at all. I did this but in the other PR.

Reply:
Did you check this component (UeerHubMobile) after changes with paddings?
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/cdd451ef-1374-48f0-b435-f58b11bd91f1)

Reply:
We have to leave overflow-x-auto because we are reducing max-height. UserHubMobile works fine, because I've added missing padding to item.

![image](https://github.com/JoinColony/colonyCDapp/assets/76940796/63336698-a346-410a-99d2-58f997392541)